VimeoNetworking/VimeoNetworking/Request+Picture.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  Request+Picture.swift
+//  Pods
+//
+//  Created by King, Gavin on 10/24/16.
+//
+//
+
+public extension Request
```

Comment:
Make sure this and the Trigger extension get added to the VimeoNetworking .xcodeproj.